solr/solr-ref-guide/src/cluster-node-management.adoc

```diff
@@ -26,6 +26,24 @@ These API commands work with a SolrCloud cluster at the entire cluster level, or
 
 Fetch the cluster status including collections, shards, replicas, configuration name as well as collection aliases and cluster properties.
 
+Additionally, this command reports a `health` status of each collection and shard, in
+order to make it easier to monitor the operational state of the collections. The
+following health state values are defined, ordered from the best to worst, based on
+the percentage of active replicas (`active`):
+
+`GREEN`::
+`active == 100%`, all replicas are active and there's a shard leader.
+`YELLOW`::
+`100% > active > 50%`, AND there's a shard leader.
+`ORANGE`::
+`50% >= active > 1%`, AND there's a shard leader.
```

Comment:
0.1 which is the logic implemented in code is 10%, not 1%. But if we change to 0.0 as suggested above, then the ORANGE rule will be `50% >= active > 0%`, and the RED rule will be `No active replicas OR there's no shard leader`.